app/main.py

```diff
@@ -1,14 +1,21 @@
 from fastapi import FastAPI
-from app.config import settings
+
 from app.db.database import init_db
+
 from app.routers import user_router
 from contextlib import asynccontextmanager
 
+from app.utils.product_setup import product_setup
+
 @asynccontextmanager
 async def lifespan(app: FastAPI):
     await init_db()
+    await startup()
```

Comment:
@juzejunior aqui tem um aparada legal que vi no docs do Fast que é o seguinte. 

ele tem 2 momentos na mesma função: antes de inicializar e após inicializar, separados por um **yield**

tipo:

await init_db()
yield
await startup()

Antes do yield roda antes do server subir, e apos o yield roda após inicializar o server. Não bloqueando a inicialização se for tarefa longa e tal. So pra saber msm.